### PR TITLE
fix wrong annotation to cases like am¹³(start with a vowel)

### PR DESCRIPTION
**Pull request by Wu-Urbanek**
My first proposal, it's not a very smart solution, but it works for now. Could you @LinguList  give me some advice on how to improve the code?

### CODE REVIEW

File: src/linse/annotate.py [hunk 1]
```diff
@@ -533,7 +533,8 @@ def seallable(
 
     if 3 <= len(sequence) <= 5:
         # first element must be the initial
-        ini = 'i' if cv[0] == 'C' else '?'
+        #ini = 'i' if cv[0] == 'C' else '?'
+        ini = 'i' if cv[0] == 'C' or cv[0]== 'V' else '?'
```
Comment (LinguList) [inline src/linse/annotate.py]: @macyl, if cv[0] is "V", there can't be an initial, the initial is nothing in this case.

**Review by LinguList — CHANGES_REQUESTED** [review]
@MacyL, thanks for trying this, but you need to think more in test cases. Please check the test file `tests/test_annotate.py`, where I added the cases we can handle now, and one case was missing: `aŋ⁵`. This resolves to `nct`, with initial and medial slot uncovered. So you should also add this test case and make sure your command can handle this, when updating the code here.

**Comment by LinguList** [comment]
If this is taking too long, and I leave you lost here, @MacyL, let me know, then I'll make the PR, and you can learn from my modifications, okay?

File: src/linse/annotate.py (hunk 1 above)
Comment (Wu-Urbanek) [reply src/linse/annotate.py]: ok, I understand. So as long as a vowel is at the start of the string, the return string is "nct". I will change it now :) (sorry, I took today as a reading day + class day).

**Comment by LinguList** [comment]
There's no rush here. Don't worry. Next week, I think, we should plan
paper number 2, so we don't lose our schedule, but you need time for
yourself as well.

**Review by LinguList — CHANGES_REQUESTED** [review]
@MacyL, do you know how to add tests for this? We also need the tests in `tests/test_annotate.py`. This also helps you to check if your code is doing what you expect it to do. If needed, I can also take over and illustrate, so you know for the next time.

**Comment by Wu-Urbanek** [comment]
I don't know how to add tests for this task, so please show me this time :)  
> @MacyL, do you know how to add tests for this? We also need the tests in `tests/test_annotate.py`. This also helps you to check if your code is doing what you expect it to do. If needed, I can also take over and illustrate, so you know for the next time.

**Comment by LinguList** [comment]
Okay, I'll try and do this tomorrow.

**Comment by LinguList** [comment]
Okay, please see my modifications. Please pull this (I updated your branch, @MacyL), and then do:

```
$ cd tests/
$ pytest test_annotate.py --cov=linse.annotate
```

This explains what part of the code is executed, which is important. 

You can also see that I added two more test pairs in test_annotate.py. These are then checked if they yield the desired result.

**Review by LinguList — APPROVED** [review]

**Pull request merged**